app/archivesspace/archivesspace.controller.js

```diff
@@ -72,10 +92,29 @@
             level: function() {
               return '';
             },
+            start_date: function() {
+              return new Date;
+            },
+            end_date: function() {
+              return new Date;
```

Comment:
Parens for the sake of my sanity wondering if `Date` and `Date()` are the same thing?